package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@elastic/synthetics",
-  "version": "1.4.0",
+  "version": "0.0.0-development",
```

Comment:
what's the reason?

Reply:
semantic-release does not update the version in version control. it only does it right before publishing.

`0.0.0-development` is an arbitrary version placeholder. It could be something else also, but this is was semantic-version sets, if you initialize it using the CLI.

https://semantic-release.gitbook.io/semantic-release/support/faq#why-is-the-package.jsons-version-not-updated-in-my-repository

there is a possibility to use the @semantic-release/git plugin to achieve this. But this would include automatic commits to main. And we don't want that.

Reply:
gotcha. Thanks for the clarification